### DISCUSSION TITLE
[FR] Search Weighting so Titles and Company match with higher value than Keywords

### DISCUSSION THREAD

**Discussion by BenParizek** [craftcms/console#96] [Craft Console] [answered]
If I currently search for "Sprout" or "Sprout Forms" our wonderful competitor Solspace appears in the results before our plugins as they appear to be ordered alphabetically:
http://share.barrelstrength.co/635c3668caa5

While, as a user, I think it's nice that related plugins come up in search results, I have no issues with firms in the store using creative keywords to appear in the results they hope to appear in, but it would be nice to have search give a higher priority to the weight of certain fields in the results when a search involves words specific to a brand, company, or plugin name.

**Comment by brianjhanson** [comment, marked as answer]
Hey @BenParizek, thanks for reaching out! We've just pushed an update to the plugin store that will sort search results by score from now on. Better late than never I hope.
